### DISCUSSION TITLE
On Key press calculate total

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6661] [Help] [answered]
Please I want to change the value of another text box as I type in another text box
As I type in the red line text box it should change the value in the Sub total and VAT
Please assist.

![image](https://user-images.githubusercontent.com/50826372/213037817-0a8eef02-cb0b-4589-8588-6e3989344e62.png)

**Comment by arunkumar12712** [comment, marked as answer]
this.form.Discount.element.keypress()

**Reply by edwardch** [reply]
Then add the code inside, like so:

```ts
this.form.Discount.element.keypress(() => {
    this.form.Total.value = (this.form.SubTotal.value * (100 - this.form.Discount.value)) * 1.185;
});
```

Make sure you calculate the total value in your save handler, rather than just on your front end...
